pkg/module/parse/template_to_module.go

```diff
@@ -140,22 +135,6 @@ func (p *Parser) newManifestFromTemplate(
 	if err != nil {
 		return nil, err
 	}
-	var componentDescriptor *compdesc.ComponentDescriptor
```

Comment:
What is the reasoning behind taking the component descriptor from the module template instead of a "true" remote one? Is it just a temporary solution?